maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -173,7 +177,17 @@ public CLIManager() {
                 .build());
         options.addOption(Option.builder(Character.toString(BATCH_MODE))
                 .longOpt("batch-mode")
-                .desc("Run in non-interactive (batch) mode (disables output color)")
+                .desc(
+                        "Run in non-interactive (batch) mode (disables output color) - alias for --non-interactive (kept for backwards compatability)")
```

Comment:
I think this is not just color. There more cases. Basically everything where standard streams aren't connected to a tty. Also blocking for input, like Maven release. Since one cannot cover all cases, I would remove the parentheses.

Reply:
Yeah, that's true. I copied the original description of --batch-mode, but I agree that it doesn't cover the functionally anymore. I've rewritten them.